src/F2_autoreport.py

```diff
@@ -566,7 +566,7 @@ def create_app(results_json):
     )
 
     # Reading the relevant user-inputs from the json_with_results.json file into Pandas dataframes
-    dfprojectData = pd.DataFrame.from_dict(results_json[PROJECT_DATA])
+    dfprojectData = pd.DataFrame.from_dict(results_json[PROJECT_DATA]).iloc[0]
```

Comment:
this is very "hidden under the carpet" what happens here, I would at least describe the rationale for the poor us in a few month

Reply:
Yeah, I tried dropping the SECTORS (which is a dict) from the results_json[PROJECT_DATA], but then I ran into another issue. I will write a description.